src/course-unit/sidebar/components/SidebarBody.jsx

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { Card, Stack } from '@openedx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import { getCourseUnitData } from '../../data/selectors';
+import { getPublishInfo } from '../utils';
+import messages from '../messages';
+import ReleaseInfoComponent from './ReleaseInfoComponent';
+
+const SidebarBody = ({ releaseLabel, isDisplayUnitLocation, locationId }) => {
+  const intl = useIntl();
+  const {
+    editedOn,
+    editedBy,
+    hasChanges,
+    publishedBy,
+    publishedOn,
+  } = useSelector(getCourseUnitData);
+
+  return (
+    <Card.Body className="course-unit-sidebar-date">
+      <Stack>
+        {isDisplayUnitLocation ? (
```

Comment:
```suggestion
        {displayUnitLocation ? (
```

I find the `is` prefix makes it harder to understand what the toggle actually does.

Reply:
Removed
Do you think the `is` prefix is not desirable in all cases or only for `displayUnitLocation`?

Reply:
Only here! :)